Eniko/index.js

```diff
@@ -0,0 +1,31 @@
+var snackServings = 120;
+var guests = 150;
+
```

Comment:
Forgot to answer  1. task. you should add snackeServings and guests and print the result!
The solution is :
console.log(snackServing + guests)